packages/manager/apps/hycu/src/routes/routes.constant.ts

```diff
@@ -1,4 +1,5 @@
 export const subRoutes = {
+  editPack: 'edit',
```

Comment:
**nitpick:** maybe call it `edit-pack` to be explicit, as you also have an `edit-name` route